models/responses.go

```diff
@@ -0,0 +1,17 @@
+package models
```

Comment:
As we split packages by logical parts and not by content type, could you please move the contents of this package to the `api` package?